packages/core-data/README.md

```diff
@@ -808,6 +859,22 @@ _Returns_
 
 -   `boolean`: Whether there is a next edit or not.
 
+### hasRevisions
```

Comment:
Do you think this is useful or using the "meta" selectors is enough?

Reply:
I blindly brought it across from the `getEntityRecord` resolver, but in my manual testing it does not appear to make any difference. 

An API call will be made with each new query, e.g., `{ _fields: 'id,date,parent' }`, and the cached query result is returned.

I can take it out for now. We can always reinstate.